bin/fetch-from-rki-metadata

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -euo pipefail
 
-curl "https://raw.githubusercontent.com/robert-koch-institut/SARS-CoV-2-Sequenzdaten_aus_Deutschland/master/SARS-CoV-2-Sequenzdaten_Deutschland.csv.xz" \
+curl "https://raw.githubusercontent.com/robert-koch-institut/SARS-CoV-2-Sequenzdaten_aus_Deutschland/master/SARS-CoV-2-Sequenzdaten_Deutschland.tsv.xz" \
```

Comment:
Use correct default branch (it's autoforwarded but this is more correct)
```suggestion
curl "https://raw.githubusercontent.com/robert-koch-institut/SARS-CoV-2-Sequenzdaten_aus_Deutschland/main/SARS-CoV-2-Sequenzdaten_Deutschland.tsv.xz" \
```

Reply:
Ah, good catch!

Reply:
FYI, in most GitHub URLs, you can `HEAD` in place of the default branch name to be more future-proof. Example:

https://raw.githubusercontent.com/robert-koch-institut/SARS-CoV-2-Sequenzdaten_aus_Deutschland/HEAD/SARS-CoV-2-Sequenzdaten_Deutschland.tsv.xz

This was discussed more in https://github.com/nextstrain/auspice/pull/1660#discussion_r1155025654